test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/https/KieServerHttpsIntegrationTest.java

```diff
@@ -116,44 +115,46 @@ public static void buildKjar() {
 
     @Test
     public void testKieServerInfo() {
-        final KieServerDeployment kieServerDeployment = deploymentScenario.getKieServerDeployments().get(0);
-        final CredentialsProvider credentialsProvider = HttpsUtils.createCredentialsProvider(kieServerDeployment.getUsername(),
-                kieServerDeployment.getPassword());
-        try (CloseableHttpClient httpClient = HttpsUtils.createHttpClient(credentialsProvider)) {
-            try (CloseableHttpResponse response = httpClient.execute(serverInforRequest(kieServerDeployment))) {
-                Assertions.assertThat(response.getStatusLine().getStatusCode()).isEqualTo(HttpsURLConnection.HTTP_OK);
-
-                final String responseContent = HttpsUtils.readResponseContent(response);
-                ServiceResponse<KieServerInfo> kieServerInfoServiceResponse = marshaller.unmarshall(responseContent, ServiceResponse.class);
-                KieServerInfo kieServerInfo = kieServerInfoServiceResponse.getResult();
-                Assertions.assertThat(kieServerInfo.getCapabilities()).contains(KieServerConstants.CAPABILITY_BRM);
+        for (final KieServerDeployment kieServerDeployment : deploymentScenario.getKieServerDeployments()) {
+            final CredentialsProvider credentialsProvider = HttpsUtils.createCredentialsProvider(kieServerDeployment.getUsername(),
+                    kieServerDeployment.getPassword());
+            try (CloseableHttpClient httpClient = HttpsUtils.createHttpClient(credentialsProvider)) {
```

Comment:
You can declare multiple `AutoCloseable` resources in a single try-with-resources to avoid this nesting:
```
try (CloseableHttpClient client = ..;
       CloseableHttpResponse response = ...) {
...
}
```

Reply:
Adjusted